src/mockUser.js

```diff
@@ -12,9 +12,11 @@ const mockUser = {
     verified_login: true,
     created_at: '2019-09-01T09:55:51.984Z',
     updated_at: '2019-11-04T11:13:14.946Z',
-    pairID: '',
+    pairID: '20',
     rank: '10',
     avatar: AVATAR_GOOGLE,
+    facebook_link: 'wwww.facebook.com',
+    phone_number: '1234235233',
```

Comment:
Phone number should be displayed this way: '123 423 523'. Please add missing spaces.

Reply:
That's a good idea. I done it as you suggested.